.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 build/
-dist/
\ No newline at end of file
+dist/
+docs/
```

Comment:
I believe `docs` need to be included to the repository in like README.md or something